tests/unit/useOnyxTest.ts

```diff
@@ -335,6 +329,12 @@ describe('useOnyx', () => {
             expect(result.current[0]).toEqual('initial value');
             expect(result.current[1].status).toEqual('loaded');
 
+            Onyx.set(ONYXKEYS.TEST_KEY, 'test1');
```

Comment:
Same: Why did you move the operation to here? Shouldn't we have tests for both cases?

Reply:
Same as above